### PR TITLE
Add guide to exporting metrics to Graphite

### PR DESCRIPTION
**Pull request by joelhans**
##### Summary

This PR creates a new guide for exporting metrics to Graphite using the exporting engine in support of the 1.23 release. Just the first of many similar guides for other exporting endpoints...

##### Component Name

area/exporting
area/docs

### CODE REVIEW

File: docs/guides/export/export-netdata-metrics-graphite.md [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+
+If you haven't already, enable the exporting engine by setting `enabled` to `yes` in the `[exporting:global]` section.
+
+```conf
+[exporting:global]
+    enabled = yes
+```
+
+Next, configure the connector. Find the `[graphite:my_graphite_instance]` example section and uncomment the line.
+Replace `my_graphite_instance` with a name of your choice&mdash;let's go with `[graphite:netdata]`. Finally, set
```
Comment (thiagoftsm) [inline docs/guides/export/export-netdata-metrics-graphite.md:89]: Do we need the `&mdsh;` here?

Comment (thiagoftsm) [reply docs/guides/export/export-netdata-metrics-graphite.md:89]: Another thing, you used `Finally, set` to start the last sentence, but the next sentence also starts with ` Finally, set the destination`. Should we use `Finally` only in the next?

Comment (joelhans) [reply docs/guides/export/export-netdata-metrics-graphite.md]: You're right on both counts. Fixed!

Comment (thiagoftsm) [reply docs/guides/export/export-netdata-metrics-graphite.md]: Thank you for the fixes.

File: docs/guides/export/export-netdata-metrics-graphite.md [hunk 2]
```diff
@@ -0,0 +1,184 @@
+<!--
+title: Export and visualize Netdata metrics in Graphite 
+description: Use Netdata to collect and export thouands of metrics to Graphite for long-term storage or further analysis.
```
Comment (vlvkobal) [inline docs/guides/export/export-netdata-metrics-graphite.md:3]: thouands -> thousands

Comment (vlvkobal) [inline docs/guides/export/export-netdata-metrics-graphite.md:3]:
```suggestion
description: Use Netdata to collect and export thousands of metrics to Graphite for long-term storage or further analysis.
```

Comment (joelhans) [reply docs/guides/export/export-netdata-metrics-graphite.md]: Done! Good catch—the one area I forgot to copy-edit.

**Pull request merged**